pkg/api/testapi/testapi.go

```diff
@@ -268,15 +266,6 @@ func init() {
 			externalTypes:        api.Scheme.KnownTypes(externalGroupVersion),
 		}
 	}
-	if _, ok := Groups[kubeadm.GroupName]; !ok {
```

Comment:
We couldn't get it working any other way, and I don't think we want to add the kubeadm API group to client-go. See the thread on Slack, please.